modules/backend/behaviors/RelationController.php

```diff
@@ -886,16 +893,15 @@ protected function makeManageWidget()
          * Form
          */
         elseif ($this->manageMode == 'form') {
-
             if (!$config = $this->makeConfigForMode('manage', 'form', false)) {
                 return null;
             }
-
+            
```

Comment:
Please remove whitespace characters from these lines. Empty lines should not have any extra whitespace characters in them